cloudpickle/cloudpickle.py

```diff
@@ -904,6 +904,12 @@ def save_root_logger(self, obj):
 
     dispatch[logging.RootLogger] = save_root_logger
 
+    if hasattr(types, "MappingProxyType"):
```

Comment:
`coverage` has a specific pragma for partial branches:
```suggestion
    if hasattr(types, "MappingProxyType"):   # pragma: no branch
```

Reply:
Indeed that seems to work on my box. Thanks your insight. I pushed a new commit to do this systematically for system-based conditions.